lib/tasks/publishing_api.rake

```diff
@@ -128,6 +128,46 @@ namespace :publishing_api do
     PublishingApiDocumentRepublishingWorker.new.perform(document.id)
   end
 
+  def ensure_document_with_content_id_doesnt_exist(content_id)
+    document = Document.find_by(content_id: content_id)
+
+    return unless document
+
+    puts "Document with this content ID exists: #{document}"
+    abort 'This Rake task is only for unknown content'
```

Comment:
Don't know if rake lint picks this up, but this seems a funny mix of single and double quotes?

Reply:
You can't use double quotes for the first string, as that's what enables using `#{ }`. You also don't need to use double quotes for the second string, as it's just a simple string.

I'm not sure there are any linting rules about using single vs double quotes...